api/fossa/issues.go

```diff
@@ -3,27 +3,39 @@ package fossa
 import (
 	"fmt"
 	"net/url"
+	"regexp"
 
 	"github.com/pkg/errors"
 )
 
 const IssuesAPI = "/api/cli/%s/issues"
 
-// An Issue holds the FOSSA API response for the issue API.
-type Issue struct {
-	ID             int
-	PriorityString string
-	Resolved       bool
-	Revision       Revision
-	Type           string
-}
-
 // A wrapped list of issues returned by the FOSSA CLI issues endpoint
 // If a push-only API key is used, then only the count is returned
 type Issues struct {
 	Count  int
 	Issues []Issue
 	Status string
+
+	NormalizedByType map[string][]Issue
+}
+
+// An Issue holds the FOSSA API response for the issue API.
+type Issue struct {
+	ID             int    `json:"id"`
```

Comment:
Doesn't specifying the JSON key this way change the output? Are no customers relying on the current names in the JSON objects? 
https://play.golang.org/p/IO7TleEfPap

Reply:
Good observation, although I am not aware of anyone who is relying on this information at the moment. The information up until now has been very limited so `fossa test` has been used as a pass-fail mechanism. Regarding this change, I am a fan of keeping the JSON consistent between the API response and what `fossa test` outputs.

Reply:
Not saying I actually know of any customers who rely on the current JSON output. You'll know better, though it would be interesting to get some insight somehow.